WebDriverAgentLib/Commands/FBOrientationCommands.m

```diff
@@ -123,4 +124,20 @@ + (NSDictionary *)_orientationsMapping
   return orientationMap;
 }
 
++ (NSDictionary *)_wdOrientationsMapping
+{
+  static NSDictionary *orientationMap;
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    orientationMap =
+    @{
```

Comment:
Can you please explain the purpose of such mapping?

Reply:
added a comment in the code